src/SFML/Graphics/Font.cpp

```diff
@@ -91,31 +92,28 @@ std::uint64_t combine(float outlineThickness, bool bold, std::uint32_t index)
 namespace sf
 {
 ////////////////////////////////////////////////////////////
-class Font::FontHandles
+struct Font::FontHandles
 {
-private:
-    // Default constructible deleter functor
-    struct Deleter
+    FontHandles() = default;
+
+    ~FontHandles()
     {
-        void operator()(FT_Library theLibrary)
-        {
-            FT_Done_FreeType(theLibrary);
-        }
-        void operator()(FT_Face theFace)
-        {
-            FT_Done_Face(theFace);
-        }
-        void operator()(FT_Stroker theStroker)
-        {
-            FT_Stroker_Done(theStroker);
-        }
-    };
+        FT_Stroker_Done(stroker);
+        FT_Done_Face(face);
+        // `streamRec` doesn't need to be explicitly freed.
+        FT_Done_FreeType(library);
```

Comment:
In code below, if initialization of some members fails, the function returns and the destructor is run -- with the remaining members default-initialized.

Is invoking each of these `FT_*` functions on `nullptr` save, with defined behavior?

Reply:
This is poorly documented, so I had to check the implementation. 

- `FT_Stroker_Done` is safe and returns void.

- `FT_Done_Face` is safe, it returns non-zero for a null argument.

- `FT_Done_FreeType` is safe, it returns non-zero for a null argument.

Reply:
Thanks a lot for checking! Could you add this as a comment, so we don't have to re-check in the future? 🙂